akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -152,6 +152,14 @@ class Flow[-In, +Out](delegate: scaladsl.Flow[In, Out]) {
    * downstream may run in parallel and may complete in any order, but the elements that
```

Comment:
so the decision was to not support supervision on user defined transform() elements now